put-a-jetpack-on-it/slides.md

```diff
@@ -0,0 +1,305 @@
+# Putting a Jetpack on your legacy codebase
+
+## Kurt Nelson
+### Android Platform at Pinterest
+
+Hey everyone, Kurt from the Pinterest Android Platform (or Android Experience) team.
```

Comment:
i do find in cool when i hear/read Androidx vs Android Experience, so something to consider